packages/server/src/cli.ts

```diff
@@ -26,7 +26,7 @@ commander.version(process.env.VERSION || "development")
 	.option("--data-dir <value>", "DEPRECATED: Use '--user-data-dir' instead. Customize where user-data is stored.")
 	.option("-h, --host <value>", "Customize the hostname.", "0.0.0.0")
 	.option("-o, --open", "Open in the browser on startup.", false)
-	.option("-p, --port <number>", "Port to bind on.", 8443)
+	.option("-p, --port <number>", "Port to bind on.", parseInt(process.env.PORT, 10) || 8443)
```

Comment:
If port is set to a non-integer it'd throw. Maybe we should parse below to offer a nice error?

Reply:
If it’s a non integer, it’d be `NaN`, and due to the `||` would fallback to `8443`